client/src/api/BackendAccess.tsx

```diff
@@ -1,16 +1,151 @@
-import axios from "axios";
-import { useRecoilValue } from "recoil";
-import { AppState } from "../AppState";
+import axios, { AxiosResponse } from "axios";
+import {
+  useMutation,
+  UseMutationResult,
+  useQuery,
+  useQueryClient,
+} from "react-query";
+import { useRecoilState, useRecoilValue } from "recoil";
+import { AppState, useFamilyMembership } from "../AppState";
+import { User } from "../parent/models/User";
+import {
+  useErrorNotification,
+  useInfoNotification,
+  useSuccessNotification,
+} from "../shell/hooks/SnackbarHooks";
+import { ChoreLoadingData } from "./models/ChoreLoadingData";
+import { QueryObject } from "./models/QueryObject";
+import { queryUrl } from "./models/QueryObjectUrl";
 
-const API_URL = process.env.REACT_APP_APIBASEURL as string;
+const apiBaseUrl = process.env.REACT_APP_APIBASEURL as string;
 
-export async function loadChores() {
-  const apiAccessToken = ""; // useRecoilValue(AppState.userBearerToken); TODO not implemented as hook
+// GET
+export function QueryGet(key: string, url: queryUrl): ChoreLoadingData {
```

Comment:
Die Hooks, wie schon erwähnt, bitte mit "use" präfixen.
Vielleicht könnte man den Namen auch verkürzen in useGet, usePost, usePut, useDelete?
Die Verwendung von Generics noch umsetzen.